internal/node/npm_package_bin.bzl

```diff
@@ -10,8 +10,10 @@ _ATTRS = {
     "args": attr.string_list(mandatory = True),
     "configuration_env_vars": attr.string_list(default = []),
     "data": attr.label_list(allow_files = True, aspects = [module_mappings_aspect, node_modules_aspect]),
+    "exit_code_out": attr.output(),
```

Comment:
`exit_code` on it's own seemed not descriptive enough, but can change these attr names to something better